api.json

```diff
@@ -0,0 +1,761 @@
+            "emailConsent": "Boolean",
+            "smsConsent": "Boolean",
+            "qrCode": "String",
+            "registeredAt": "String",
+            "lastActiveAt": "String"
+          }
+        },
+        "POST": {
+          "description": "Create a new user",
+          "auth": true,
+          "scope": [],
+          "query": {
+            "name": "String",
+            "regionId": "Number",
+            "sectorId": "Number",
+            "organisationId": "Number",
+            "gender": "String",
+            "birthYear": "String",
+            "email": "String",
+            "phoneNumber": "String",
+            "emailConsent": "Boolean",
+            "smsConsent": "Boolean",
+            "qrCode": "String"
+          },
+          "response": {
+            "id": "Number",
+            "name": "String",
+            "regionId": "Number",
+            "sectorId": "Number",
+            "organisationId": "Number",
+            "gender": "String",
+            "birthYear": "String",
+            "email": "String",
+            "phoneNumber": "String",
+            "emailConsent": "Boolean",
+            "smsConsent": "Boolean",
+            "qrCode": "String",
+            "registeredAt": "String",
+            "lastActiveAt": "String"
+          }
+        },
+        "PUT": {
+        "description": "Update details of a specific user",
+        "auth": true,
+        "scope": [],
+        "query": {
+          "name": "String",
+          "regionId": "Number",
+          "sectorId": "Number",
+          "organisationId": "Number",
+          "gender": "String",
+          "birthYear": "String",
+          "email": "String",
+          "phoneNumber": "String",
+          "emailConsent": "Boolean",
+          "smsConsent": "Boolean",
+          "qrCode": "String",
+          "delete": "Boolean"
+        },
+        "response": {
+          "id": "Number",
+          "name": "String",
+          "regionId": "Number",
+          "sectorId": "Number",
+          "organisationId": "Number",
+          "gender": "String",
+          "birthYear": "String",
+          "email": "String",
+          "phoneNumber": "String",
+          "emailConsent": "Boolean",
+          "smsConsent": "Boolean",
+          "qrCode": "String",
+          "registeredAt": "String",
+          "lastActiveAt": "String"
+        }
+      },
+      "/:id/password/reset": {
+        "POST": {
+          "description": "Reset password for a single user",
+          "auth": true,
+          "scope": [],
+          "body": {
+            "userId": "Number",
+            "email": "String",
+            "password": "String",
+            "confirmPassword": "String"
+          },
+          "response": {}
+        }
+      }
+      },
+     "/:id/visit_activities/:id": {
+      "POST": {
+        "description": "",
+        "auth": true,
+        "scope": [],
+        "query": {},
+        "response": {}
+      }
+      },
+      "/:id/volunteers": {
```

Comment:
I think naming should be clearer here. We're not retrieving volunteers, we're retrieving volunteering logs. `/:id/volunteer_logs` is better IMO.